docs/contributing/designer.rst

```diff
@@ -0,0 +1,109 @@
+Designer
+########
+
+.. vale off
+
+Hello, designer. Welcome to the Mautic Community!
+
+.. vale on
+
+Thank you for considering contributing to Mautic. Currently, there aren’t many designers actively contributing to Mautic, so this is a great opportunity for you to make a big difference.
+
+As a designer, you can contribute in various ways, depending on your interests and how you can apply your skills to current projects.
+
+Some areas you may want to consider:
+
+.. vale off
+
+- Supporting the Product Team by creating visual resources to explain new features and improving the user experience of Mautic
+- Helping the Marketing Team with designing their monthly newsletters
+- Creating new designs for Mautic's web resources (for example, the :xref:`Mautic website`)
+- Contributing to Strategic Initiatives by designing the user experience for new features
+- Improving the user documentation by creating illustrations or images that complement the written information
+- Building a pattern library to standardize the UI patterns used in Mautic
+- Joining the User Experience Tiger Team
+
+Getting started
```

Comment:
```suggestion
Getting started as a designer
```

All headings need to be unique, so let's make this specific to designers (please add the extra underlines, too!)

Reply:
Done! 😊